### PR TITLE
Add intro video toggle buttons with minimal styling

### LINKED ISSUE #227: Add intro videos part 1, protocol, part 2 (with AL)
Acceptance

- [ ] Press button to see Welcome, Protocol or Next Steps video
- [ ] I can start playing without viewing any of the videos
- [ ] I can go between the videos

Nice to have:
- [ ] It is clear which order I should play the videos in
- [ ] Intro part 1 autoplays

### PR DESCRIPTION
**Pull request by katjam**
Fixes #227

## Description

- Adds buttons to toggle between 3 intro videos
- Could use improved styling for UX

@TheLabCollective/developers

### CODE REVIEW

File: src/Video.elm [hunk 1]
```diff
@@ -15,13 +15,13 @@ videoToData video =
             { id = "oWzsFH4LLME", title = "Let’s get started - YouTube" }
 
         Intro1 ->
-            { id = "VB0y52wzNZU", title = "" }
+            { id = "mvo5p86uYNc", title = "Welcome to Biocore - YouTube" }
 
         Intro2 ->
-            { id = "rztd_BIXyUQ", title = "" }
+            { id = "zDvg0tnHc0Q", title = "Protocol - YouTube" }
```
Comment (Nikomus) [inline src/Video.elm:21]: This one seems to be blocked from embedding, have asked for it to be allowed.

File: src/View/Intro.elm [hunk 2]
```diff
@@ -4,20 +4,46 @@ import Copy.Keys exposing (Key(..))
...
+                , href (Route.toString Route.Desktop)
+                ]
+                [ text (t StartNewGame) ]
             ]
         ]
+
+
+renderVideoButtons : Html Msg
+renderVideoButtons =
+    ol [ class "row my-4 list-unstyled justify-content-md-center" ]
```
Comment (Nikomus) [inline src/View/Intro.elm:34]: Probably want `max-width: 800px` on here to match the video above. I would usually put `display: flex` on the list and switch from `flex-direction: row` on desktop to `flex-direct: column` on mobile.

File: src/View/Intro.elm [hunk 3]
```diff
@@ -4,20 +4,46 @@ import Copy.Keys exposing (Key(..))
...
+        (List.map
+            (\( buttonTextKey, videoKey ) ->
+                li [ class "col text-center" ]
+                    [ button
```
Comment (Nikomus) [inline src/View/Intro.elm:38]: type="button"?

**Review by Nikomus — APPROVED** [review]
Maybe a bit confusing that the active video has a watch button, but this looks good to get all the videos in 📹

File: src/Video.elm (hunk 1 above)
Comment (katjam) [reply src/Video.elm:21]: I sent Bentley & co an email earlier - and am going to double check all of them from the dropbox when I do: #238

File: src/View/Intro.elm (hunk 3 above)
Comment (katjam) [reply src/View/Intro.elm:38]: Does it need that as well?

File: src/View/Intro.elm (hunk 2 above)
Comment (katjam) [reply src/View/Intro.elm:34]: Is there a good bootstrap way to do that?

File: src/View/Intro.elm (hunk 3 above)
Comment (Nikomus) [reply src/View/Intro.elm:38]: The default if not specified is submit, which this isn't doing. Not a big problem here outside a form.

Comment (katjam) [reply src/View/Intro.elm:38]: Ah - yeah - totally forgot that. Thanks.

**Comment by katjam** [comment]
@Nikomus I made some small style improvements

**Review by Nikomus — APPROVED** [review]
New style bits look great! 💃

**Pull request merged**